docs/source/onnx.rst

```diff
@@ -598,6 +598,9 @@ Functions
 .. autofunction:: register_custom_op_symbolic
 .. autofunction:: select_model_mode_for_export
 .. autofunction:: is_in_onnx_export
+.. autofunction:: onnx_logging_enabled
+.. autofunction:: set_onnx_logging_enabled
```

Comment:
```suggestion
.. autofunction:: config_log
```
This setter does more than just enabling/disabling the log, so a more comprehensive name would be more intuitive. 

`(Python's logging module)[https://docs.python.org/3/library/logging.html#logging.basicConfig]` exposes the method `basicConfig`. Maybe we could do something similar such as `torch.onnx.config_log` to specify both status and/or stream.

Having `torch.onnx.enable_log()`, `torch.onnx.disable_log()` and `torch.onnx.set_stream(stream_name: str)` could be also intuitive and easy to use

Reply:
Thank you for suggestions! I also like the second suggestion, which is very clean. I want to keep it simple that way.